serve.go

```diff
@@ -158,6 +177,22 @@ func connectorLogLevel(cfg pconnector.PluginConfig) zerolog.Level {
 	return l
 }
 
+// connectorGRPCMaxReceiveRecordSize returns the maximum size of a record that can be received
+// by the connector. The value is fetched from the environment variable provided by conduit-connector-protocol.
+func connectorGRPCMaxReceiveRecordSize() int {
+	maxReceiveRecordSize := os.Getenv(pconnector.EnvConduitConnectorMaxReceiveRecordSize)
+	if maxReceiveRecordSize == "" {
+		return server.DefaultMaxReceiveRecordSize
+	}
```

Comment:
Nitpick: I think this `if` is redundant, right? If we try to parse an empty string we'll get an error and return the default anyway 🤷